galaxy_ng/app/tasks/index_registry.py

```diff
@@ -0,0 +1,160 @@
+import json
+import logging
+from urllib.parse import quote, urlencode
+
+from django.core import exceptions
+from django.utils.translation import gettext_lazy as _
+from django.http.request import HttpRequest
+
+from pulpcore.plugin.tasking import dispatch
+
+from pulp_container.app import models as container_models
+
+from galaxy_ng.app.api.ui import serializers
+from galaxy_ng.app import models
+
+
+log = logging.getLogger(__name__)
+
+
+CATALOG_API = "https://catalog.redhat.com/api/containers/v1/repositories"
+
+
+class CouldNotCreateContainerError(Exception):
+    def __init__(self, remote_name, error=""):
+        self.message = _(
+            f"Failed to create container {remote_name}. {error}"
```

Comment:
IIRC we should not use `f".."` inside a `_()` translation.
```suggestion
            _("Failed to create container {remote_name}. {error}").format(remote_name=remote_name, error=error)
```

@jerabekjiri can confirm

Reply:
> IIRC we should not use f".." inside a _() translation

that is correct